site/en/guides/build/logs.md

```diff
@@ -220,14 +218,14 @@ For example, to enable this for an nrf52840 connected to a Linux host:
 
 1.  Edit the configuration file for the platform and change the log output to
     NCP Spinel. For nrf52840, this is
-    `/examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h`:
+    `/src/nrf52840/openthread-core-nrf52840-config.h`:
```

Comment:
```suggestion
    `./src/nrf52840/openthread-core-nrf52840-config.h` in [ot-nrf528xx](https://github.com/openthread/ot-nrf528xx) repository:
```

Reply:
Building off @librasungirl 's suggestion, 

in the [ot-nrf528xx]